custom_components/plugwise/sensor.py

```diff
@@ -8,11 +8,12 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from plugwise.nodes import PlugwiseNode
 
-from .const import CB_NEW_NODE, COORDINATOR, DOMAIN, LOGGER, PW_TYPE, STICK, USB
+from .const import CB_NEW_NODE, COORDINATOR, DOMAIN, LOGGER, STICK, USB  # pw-beta usb
```

Comment:
Not all imported constants are used by USB only.

Reply:
Whole lot of love goes to isort :(